README.md

```diff
@@ -27,3 +28,9 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+### Invariant explanation
+The invarient that must be true at the beginning of every recursive call is that n must be greater than 1.
+This is because it keeps calling the function until n == 1 and that starts the array with [0, 1] and
+continues to add to that until it adds the n'th term to the end of the array. Otherwise, if the initial case
+is n == 1 or n == 0, there is no recursion necessary. This makes sure that we always have the correct n'th number because we will always have the correct starting numbers. The most important part of the fibonacci sequence is to have the 0th and 1st numbers because the rest of the series is based off of that. n being greater than 1 at the beginning of the current iteration tells us that we will need to make another recursive call because we have not reached the case where we can start with the correct numbers.
```

Comment:
But what do you eventually know about the state of the array in order to be able to compute the current Fibonacci number?

Reply:
I must not understand what you're looking for. Do you mean that we eventually know that the n-1 and n-2 numbers are correct making it possible for the nth number to be correct?

Reply:
Yes.

Reply:
Ok thanks, I have added that to the end of my readme.